Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.4
+
+RUN apt-get update && apt-get install build-essential pkg-config cmake -y
+
+ENV PKG /go/src/github.com/docker/docker-network
+
+# Get libgit2
+ENV LIBGIT2=github.com/tiborvass/git2go
+ENV LIBGIT2_ORIG=github.com/libgit2/git2go
+RUN go get -d ${LIBGIT2} && \
```

Comment:
Also we can clone `go_backends` right here with `-b`. We can even reduce time of clone with `--single-branch`